sensors/nxt_i2c_sensor_defs.c

```diff
@@ -2044,6 +2044,83 @@ const struct nxt_i2c_sensor_info nxt_i2c_sensor_defs[] = {
 			},
 		}
 	},
+	[MS_IR_THERMOMETER] = {
+		/**
+		 * @vendor_name: mindsensors.com
+		 * @vendor_part_number: IRThermometer
+		 * @vendor_part_name: IRThermometer
+		 * @vendor_website: http://www.mindsensors.com/products/170-ir-temperature-sensor-for-ev3-or-nxt
+		 * @default_address: 0x2A
+		 */
+		.name		= MS_IR_THERMOMETER_NAME,
```

Comment:
`MS_IR_THERMOMETER_NAME` needs to be added to the list [here](https://github.com/ev3dev/lego-linux-drivers/blob/ev3dev-stretch/sensors/nxt_i2c_sensor.h#L162)